src/pages/Users/helper.tsx

```diff
@@ -0,0 +1,25 @@
+export const FittingDataUserPage = (usersDataResponse) => {
+   const tableData = {
+    header:'Users',
+    data: {
+     headers: [],
+     data: []
+   }
+   }
+
+
+  Object.keys(usersDataResponse[0]).forEach((key) => {
+    const title = key.split('')
+    title[0] = title[0].toUpperCase()
+    const header = {
+      title: title.join(''),
+      property: key
+    }
+    tableData.data.headers.push(header)
```

Comment:
I guess here array's `map` can be used also

Reply:
done